sample-app/ios/Podfile

```diff
@@ -1,13 +1,45 @@
 # Uncomment the next line to define a global platform for your project
 platform :ios, '11.0'
+require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
 target 'sample-app' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   # use_frameworks!
-
+  pod 'NetLogger'
```

Comment:
no, we don't want this in the sample app

Reply:
👍